src/app/products/products-routing.module.ts

```diff
@@ -219,17 +221,30 @@ const routes: Routes = [
           path: 'recurring-deposit-products',
           component: RecurringDepositProductsComponent,
           resolve: {
-                recurringDepositProducts: RecurringDepositProductsResolver
+            recurringDepositProducts: RecurringDepositProductsResolver
```

Comment:
Remove all these corrections to indentation from this PR. This should not be a part of this.